schemas/firefox-accounts/activity-flow-metrics/activity-flow-metrics.1.schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "document_id": {
+      "description": "UUIDv4 message identifier, normally parsed out of the uri by the ingestion edge service.",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+      "type": "string"
+    },
+    "document_namespace": {
+      "description": "Per-project document namespace, normally parsed out of the uri by the ingestion edge service.",
+      "enum": [
+        "firefox-accounts"
+      ],
+      "type": "string"
+    },
+    "document_type": {
+      "description": "Per-project document type, matching the schema title, normally parsed out of the uri by the ingestion edge service.",
+      "enum": [
+        "activity-flow-metrics"
+      ],
+      "type": "string"
+    },
+    "document_version": {
+      "description": "Document version number, normally parsed out of the uri by the ingestion edge service.",
+      "enum": [
+        "1"
+      ],
+      "type": "string"
+    },
+    "submission_timestamp": {
+      "description": "Time when the Firefox Accounts web server received this message, formatted as ISO time string.",
+      "format": "date-time",
+      "type": "string"
+    }
+  },
+  "required": [
+    "document_id",
+    "document_namespace",
+    "document_type",
+    "document_version",
+    "submission_timestamp"
+  ],
+  "title": "activity-flow-metrics",
+  "type": "object"
+}
```

Comment:
Since there are currently no non-metadata fields in this schema, the schema can be just:

```
{
  "$schema" : "http://json-schema.org/draft-04/schema#",
   "title": "activity-flow-metrics",
  "type" : "object",
  "title" : "placeholder_schema",
  "properties" : {
  }
}
```